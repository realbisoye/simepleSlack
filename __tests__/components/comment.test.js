import 'react-native'
import React from 'react'
import Comment from '../../app/components/comment'
import renderer from 'react-test-renderer'

it('renders comment component correctly', () => {
  const time = new Date('April 6, 2019 23:15:30 GMT+00:00').toLocaleTimeString('en-US', {
    hour: '2-digit',
    minute: '2-digit',
  })
  const rendered = renderer.create(
    <Comment message={'Hi'} time={time}/>
  )
  expect(rendered).toMatchSnapshot()
})
