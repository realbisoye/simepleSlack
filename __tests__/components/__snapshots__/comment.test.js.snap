// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders comment component correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "paddingVertical": 5,
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../app/assets/slackbot.png",
      }
    }
    style={
      Object {
        "height": 50,
        "width": 50,
      }
    }
  />
  <View
    style={
      Object {
        "flexDirection": "column",
        "marginLeft": 5,
        "marginTop": 7,
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
          "marginBottom": 3,
        }
      }
    >
      <Text
        style={
          Object {
            "fontWeight": "bold",
            "marginRight": 5,
          }
        }
        testID="appTitle"
      >
        John Legend
      </Text>
      <Text
        style={
          Object {
            "color": "#808080",
          }
        }
        testID="appTitle"
      >
        7:15 PM
      </Text>
    </View>
    <Text
      style={
        Object {
          "color": "#404040",
        }
      }
    >
      Hi
    </Text>
  </View>
</View>
`;
